www/class/centreonSession.class.php

```diff
@@ -86,7 +86,7 @@ public function registerVar($registerVar)
         }
     }
 
-    public function checkSession($sessionId, $pearDB)
+    public static function checkSession($sessionId, $pearDB)
```

Comment:
It is not advisable to change a non-static call to a static call.
Even if PHP allows the non-static call of static methods, this makes no sense in object programming.